contrib/lib/src/templates/mod.rs

```diff
@@ -76,6 +78,11 @@ const DEFAULT_TEMPLATE_DIR: &'static str = "templates";
 /// [Serde](https://github.com/serde-rs/json) and would serialize to an `Object`
 /// value.
 ///
+/// In debug mode (without the `--release` flag passed to `cargo`), templates
+/// will be automatically reloaded from disk if any changes have been made to
+/// the templates directory since the previous request. In release builds,
+/// template reloading is unavailable for better performance.
```

Comment:
If you like this wording, I can add something similar to the guide.

Reply:
I do like this wording, though I'd say "disabled" instead of "unavailable". I'd also make it clear that there's no way to enable it.